CHANGELOG.md

```diff
@@ -15,6 +16,8 @@
 
 Contributors:
 - [@NiallRees](https://github.com/NiallRees) ([#4447](https://github.com/dbt-labs/dbt-core/pull/4447))
+- [@alswang18](https://github.com/alswang18) ([#4644]
+https://github.com/dbt-labs/dbt-core/pull/4644))
```

Comment:
```suggestion
- [@alswang18](https://github.com/alswang18) ([#4644](https://github.com/dbt-labs/dbt-core/pull/4644))
```

Reply:
Just fixing the broken link and then this LGTM!

Reply:
Added the change :)